mitodl/unified-ecommerce
# Adds infrastructure containers to the app

### What are the relevant tickets?

https://github.com/mitodl/hq/issues/3166
https://github.com/mitodl/hq/issues/3158

### Description (What does it do?)

Just local infrastructure stuff:

- Adds a sample APISIX container (note that this is not completely configured)
- Adds a sample Traefik container (this is set up)
- Updates the README to discuss how these things work
- Moves the app's nginx config so the config folder is organized

This does not include any of the software support for the app to use the API gateways.

### How can this be tested?

You should be able to build and run the app as per usual.

You should be able to run the Traefik environment in its own project (see the README); you should then be able to get to the app via the Traefik proxy. Note that auth won't be here so it'd be easiest to just try to hit the Swagger UI.
